src/patient-registration/input/custom-input/relationship-input.component .tsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState, useEffect, Fragment } from 'react';
+import styles from './../input.css';
+import { SelectInput } from './select-input.component';
+import { getAllRelationshipTypes, getPerson } from '../../patient-registration.resource';
+import { createErrorHandler } from '@openmrs/esm-error-handling';
+import { FieldArray } from 'formik';
+import Autosuggest from 'react-autosuggest';
+//import styles from './autosuggest.css';
+
+interface RelationshipInputProps {
+  setFieldValue(field: string, value: any, shouldValidate?: boolean): void;
```

Comment:
Is it possible to have a better typing for `value: any`?

Reply:
Once this is decided on then we need to refactor other components receiving this prop as well